exonum/src/blockchain/mod.rs

```diff
@@ -287,15 +293,15 @@ impl BlockchainMut {
         }
 
         // Get tx & state hash.
-        let schema = Schema::new(&fork);
+        let schema = Schema::get_unchecked(&fork);
```

Comment:
`Schema::get_unchecked` is called 30 times within that PR, and `Schema::get` is only 4. I don't know why, but it just don't feel right.